source/_coverage.erb

```diff
@@ -0,0 +1,7 @@
+<ol class="coverage">
+  <% coverage_links.each do |coverage_link| %>
+    <li class="coverage-item <%= coverage_link.type %>">
+      <a href="<%= coverage_link.url %>" rel="nofollow"><%= coverage_link.title %></a>
```

Comment:
Do we need escaping in here? feesls like title could easily contain & < etc. no idea if this happens automatically in middleman

Reply:
Good point, I'll check as I _think_ there's a talk with some erb tags in the title.  I'd _hope_ that the middleman (I think it uses padrino helpers) does escaping as I don't want to go back to the days of having to `h(..)` or `.html_safe` all my strings.